manim/mobject/mobject.py

```diff
@@ -1011,9 +1011,11 @@ def scale(self, scale_factor: float, **kwargs) -> "Mobject":
             return self
 
     def rotate_about_origin(self, angle, axis=OUT, axes=[]):
+        """Rotates an mobject about the ORIGIN, which is at [0,0,0]"""
         return self.rotate(angle, axis, about_point=ORIGIN)
 
     def rotate(self, angle, axis=OUT, **kwargs):
+        """Rotates an mobject about a certain point e.g. `m.rotate(PI, about_point = 2*LEFT)`"""
```

Comment:
The last part is here not rendered in the docs:
![image](https://user-images.githubusercontent.com/44469195/113145009-33320880-922e-11eb-86c5-7473bc384e09.png)
Do you know why?
Maybe to long?

Reply:
I'll try that.